--- patch-a/src/basic/filesystems-gperf.gperf	2026-05-04 10:43:05.313007664 +0200
+++ patch-b/src/basic/filesystems-gperf.gperf	2026-05-04 10:43:05.322973354 +0200
@@ -29,6 +29,7 @@
 autofs,          {AUTOFS_SUPER_MAGIC}
 balloon-kvm,     {BALLOON_KVM_MAGIC}
 bdev,            {BDEVFS_MAGIC}
+bcachefs,        {BCACHEFS_SUPER_MAGIC}
 binder,          {BINDERFS_SUPER_MAGIC}
 binfmt_misc,     {BINFMTFS_MAGIC}
 bpf,             {BPF_FS_MAGIC}
@@ -68,6 +69,7 @@
 # gfs is an old version of gfs2 and reuses the magic
 gfs,             {GFS2_MAGIC}
 gfs2,            {GFS2_MAGIC}
+guestmemfd,      {GUEST_MEMFD_MAGIC}
 hostfs,          {HOSTFS_SUPER_MAGIC}
 hpfs,            {HPFS_SUPER_MAGIC}
 hugetlbfs,       {HUGETLBFS_MAGIC}
@@ -85,12 +87,14 @@
 nfs4,            {NFS_SUPER_MAGIC}
 nilfs2,          {NILFS_SUPER_MAGIC}
 nsfs,            {NSFS_MAGIC}
+nullfs,          {NULL_FS_MAGIC}
 ntfs,            {NTFS_SB_MAGIC}
 ntfs3,           {NTFS3_SUPER_MAGIC}
 ocfs2,           {OCFS2_SUPER_MAGIC}
 openpromfs,      {OPENPROM_SUPER_MAGIC}
 orangefs,        {ORANGEFS_DEVREQ_MAGIC}
 overlay,         {OVERLAYFS_SUPER_MAGIC}
+pidfs,           {PID_FS_MAGIC}
 pipefs,          {PIPEFS_MAGIC}
 ppc-cmm,         {PPC_CMM_MAGIC}
 proc,            {PROC_SUPER_MAGIC}
