--- patch-a/meson.build	2026-05-04 10:43:05.326597521 +0200
+++ patch-b/meson.build	2026-05-04 10:43:05.330755803 +0200
@@ -373,7 +373,7 @@
         '-Werror=missing-declarations',
         '-Werror=missing-prototypes',
         '-Werror=overflow',
-        '-Werror=override-init',
+        '-Wno-error=override-init',
         '-Werror=return-type',
         '-Werror=shift-count-overflow',
         '-Werror=shift-overflow=2',
